argilla/docs/getting_started/quickstart.md

```diff
@@ -59,6 +88,7 @@ pip install argilla
 ```
 
 ## Create your first dataset
```

Comment:
We could actually just lose this section, and reference the guide on creating datasets in the SDK. 🤔